docs/topics/i18n/timezones.txt

```diff
@@ -231,9 +238,8 @@ When you enable time zone support, Django interprets datetimes entered in
 forms in the :ref:`current time zone <default-current-time-zone>` and returns
 aware datetime objects in ``cleaned_data``.
 
-If the current time zone raises an exception for datetimes that don't exist or
-are ambiguous because they fall in a DST transition (the timezones provided by
-pytz_ do this), such datetimes will be reported as invalid values.
+Converted datetimes that don't exist or are ambiguous because they fall in a
```

Comment:
I can see from the tests that this is the case (except for one situation, which is marked as a bug 😅), but is there some advice for how people are supposed to handle this situation? Probably non-existent times won't be an issue, but ambiguous times are real, just under-specified.

Is the idea that for a form in `Europe/Paris`, you'd want to specify, e.g. `'2011-10-30 02:30:00'` as either `'2011-10-30 02:30:00+01:00'` or `'2011-10-30 02:30:00+02:00'`? It seems to defeat the purpose of specifying "local time + zone" if the end user needs to know the offset in a zone at a specific time. Presumably the best thing to do would be to allow the person creating the form (possibly even via user input) to specify a strategy for what to do in the case of ambiguous datetimes?

I suppose this is just an aside, though, since the actual underlying behavior is not changed in this particular PR.

Reply:
I'm going to pass on this point for now. We don't really have a story, except earlier on there's a _In practice, this is rarely an issue._ … 

I will ponder it going forward though. 

Thanks.